tequilapi/endpoints/service.go

```diff
@@ -116,14 +116,14 @@ func (se *ServiceEndpoint) ServiceList(resp http.ResponseWriter, _ *http.Request
 //       "$ref": "#/definitions/ErrorMessageDTO"
 func (se *ServiceEndpoint) ServiceGet(resp http.ResponseWriter, _ *http.Request, params httprouter.Params) {
 	id := service.ID(params.ByName("id"))
-	service := se.serviceManager.Service(id)
 
-	if service == nil {
-		utils.SendErrorMessage(resp, "Requested service not found", http.StatusNotFound)
+	serviceInstance := se.serviceManager.Service(id)
+	if serviceInstance == nil {
+		utils.SendErrorMessage(resp, "Requested serviceInstance not found", http.StatusNotFound)
```

Comment:
Isn't `Requested service not found` was more human readable?

Reply:
It was bulk renaming typo